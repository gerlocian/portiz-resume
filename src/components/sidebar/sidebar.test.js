'use strict';

import React from 'react';
import ReactTestUtils from 'react-dom/test-utils';
import Renderer from 'react-test-renderer';
import SideBar from './sidebar.component';

describe('Sidebar', () => {
    it('should be a react component', () => {
        expect(ReactTestUtils.isCompositeComponentElement(
            <SideBar/>
        )).toBe(true);
    });

    it('should match the snapshot', () => {
        const component = Renderer.create(<SideBar/>).toJSON();
        expect(component).toMatchSnapshot();
    });
});