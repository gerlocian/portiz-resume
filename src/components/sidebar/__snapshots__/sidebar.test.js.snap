// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar should match the snapshot 1`] = `
<div
  className="sidebar "
>
  <div
    className="hamburger-icon"
  >
    <a
      href="#"
      onClick={[Function]}
    >
      <i
        className="material-icons"
      >
        menu
      </i>
    </a>
  </div>
  <div
    className="backdrop"
    onClick={[Function]}
  />
  <aside
    className="sidebar-content"
  />
</div>
`;
